### PR TITLE
PAR-1142: Member cease journey.

### PR DESCRIPTION
**Pull request by kalpaitch**

### CODE REVIEW

File: web/modules/custom/par_forms/src/Element/GdsDate.php [hunk 1]
```diff
@@ -44,7 +44,6 @@ public function getInfo() {
         [$class, 'preRenderCompositeFormElement'],
       ],
       '#date_date_format' => 'Y-m-d',
-      '#theme_wrappers' => ['form_element'],
```
Comment (stemount) [inline web/modules/custom/par_forms/src/Element/GdsDate.php]: is this a mistake?

**Comment by stemount** [comment]
Looks good, just a rogue line of code maybe?

File: web/modules/custom/par_forms/src/Element/GdsDate.php (hunk 1 above)
Comment (kalpaitch) [reply web/modules/custom/par_forms/src/Element/GdsDate.php]: It was repeating the '#title' and description wrappers. Not sure why or if there's a better way.

Comment (stemount) [reply web/modules/custom/par_forms/src/Element/GdsDate.php]: No, I reckon this is not needed, preRenderCompositeFormElement adds a fieldset, so 👍

**Pull request merged**